controllers/operandrequest/operandrequest_controller_test.go

```diff
@@ -162,17 +182,31 @@ var _ = Describe("OperandRegistry controller", func() {
 
 			By("Disabling the etcd operator")
 			Eventually(func() bool {
-				requestInstance := &operatorv1alpha1.OperandRequest{}
-				Expect(k8sClient.Get(ctx, requestKey, requestInstance)).Should(Succeed())
-				requestInstance.Spec.Requests[0].Operands = requestInstance.Spec.Requests[0].Operands[1:]
-				Expect(k8sClient.Update(ctx, requestInstance)).Should(Succeed())
+				requestInstance1 := &operatorv1alpha1.OperandRequest{}
+				Expect(k8sClient.Get(ctx, requestKey1, requestInstance1)).Should(Succeed())
+				requestInstance1.Spec.Requests[0].Operands = requestInstance1.Spec.Requests[0].Operands[1:]
+				Expect(k8sClient.Update(ctx, requestInstance1)).Should(Succeed())
+				requestInstance2 := &operatorv1alpha1.OperandRequest{}
+				Expect(k8sClient.Get(ctx, requestKey2, requestInstance2)).Should(Succeed())
+				requestInstance2.Spec.Requests[0].Operands = requestInstance2.Spec.Requests[0].Operands[1:]
+				Expect(k8sClient.Update(ctx, requestInstance2)).Should(Succeed())
 				etcdCluster := &v1beta2.EtcdCluster{}
 				err := k8sClient.Get(context.TODO(), types.NamespacedName{Name: "example", Namespace: operatorNamespaceName}, etcdCluster)
 				return err != nil && errors.IsNotFound(err)
 			}, testutil.Timeout, testutil.Interval).Should(BeTrue())
 
-			By("Deleting the OperandRequest")
-			Expect(k8sClient.Delete(ctx, request)).Should(Succeed())
+			By("Deleting the first OperandRequest")
+			Expect(k8sClient.Delete(ctx, request1)).Should(Succeed())
+
+			By("Checking jenkins operator have not been deleted")
+			Eventually(func() bool {
+				jenkinsCSV := &olmv1alpha1.ClusterServiceVersion{}
+				err := k8sClient.Get(ctx, types.NamespacedName{Name: "jenkins-csv.v0.0.1", Namespace: operatorNamespaceName}, jenkinsCSV)
+				return err == nil
+			}, testutil.Timeout, testutil.Interval).Should(BeFalse())
```

Comment:
For this unit test, I tried to check whether jenkins operator is not deleted, the correct behavior should be `true`, and both `Should(BeFalse())` and `Should(BeTrue())` pass the test.

@horis233

Reply:
hmmm....Could you take a look at the comment below, thank you! @horis233 
https://github.com/IBM/operand-deployment-lifecycle-manager/pull/639#discussion_r599193928